doc/source/math/geocoordinates.rst

```diff
@@ -107,40 +107,40 @@ Structure
 
     The ship-raw 3D position above or below the surface of the body, relative to the current ship's Center of mass.  You pass in an altitude number for the altitude above "sea" level of the desired location.
 
-Examples Usage
---------------
+Example Usage
+-------------
 
 ::
 
-    SET spot TO LATLNG(10, 20).     // Initialize point at latitude 10,
-                                    // longitude 20
+    // Initialize point at latitude 10, longitude 20:
+    SET spot TO LATLNG(10, 20).
                                     
-    PRINT spot:LAT.                 // Print 10
-    PRINT spot:LNG.                 // Print 20
+    PRINT spot:LAT.     // Print 10
+    PRINT spot:LNG.     // Print 20
     
-    PRINT spot:DISTANCE.            // Print distance from vessel to x
-                                    // (same altitude is presumed)
-    PRINT spot:HEADING.             // Print the heading to the point
-    PRINT spot:BEARING.             // Print the heading to the point
-                                    // relative to vessel heading
+    // Print distance from vessel to x (same altitude is presumed):
+    PRINT spot:DISTANCE.
+    // Print the heading to the point
+    PRINT spot:HEADING.
+    // Print the heading to the point relative to vessel heading:
+    PRINT spot:BEARING.
                                     
-    SET spot TO SHIP:GEOPOSITION.   // Make spot into a location on the
-                                    // surface directly underneath the
-                                    // current ship
+    // Make spot into a location on the surface directly underneath the current ship:
+    SET spot TO SHIP:GEOPOSITION.
                                     
-    SET spot TO LATLNG(spot:LAT,spot:LNG+5). // Make spot into a new
-                                             // location 5  degrees east
-                                             // of the old one
+    // Make spot into a new location 5 degrees east of the old one:
+    SET spot TO LATLNG(spot:LAT,spot:LNG+5).
```

Comment:
```
line 1 // comment A
line 2
line 3
```
Meaning: "Line 1 is explained with comment A, while line 2 and line 3 are not commented."
```
// comment A:
line 1
line 2
line 3
```
Meaning: "comment A refers to the next upcoming block of code.  Maybe it's just referring to line 1, or maybe it's referring to the work being done collectively by lines 1, 2, and 3.  You don't know.

If there's enough room to allow for it, I prefer to put the comment at the end if it refers to just that one line only, and put it as a header on its own line only when it's referring to a multi-line chunk of code, or if it's just too long to fit (in which case I'll try to separate with vertical space after the next line to make it clear that it only refers to the next one single line - but doing this too much makes the code take lots of screen space to just say a few lines because of all the extra blank lines this requires.

But really the reason for this is because I was deliberately *avoiding* consistency in comment usage from one example to the next.  I wanted to make sure there were examples of different valid syntax formats in use throughout the examples in the docs.  When *every single example you ever see* in the manual always formats things the same way, and it's the manual that is explaining the syntax to you, it muddies the explanation of how the syntax really works.